samples/gui_test_wgpu/src/gui_test_wgpu.zig

```diff
@@ -478,12 +489,30 @@ fn update(demo: *DemoState) !void {
                 var si8: i8 = 0;
                 var v3u8: [3]u8 = .{ 0, 0, 0 };
             };
-            _ = zgui.inputText("Input text", .{ .buf = static.buf[0..] });
-            _ = zgui.inputTextMultiline("Input text multiline", .{ .buf = static.buf1[0..] });
-            _ = zgui.inputTextWithHint(
-                "Input text with hint",
-                .{ .hint = "Enter your name", .buf = static.buf2[0..] },
-            );
+            zgui.separatorText("static input text");
+            _ = zgui.inputText("Input text", .{ .buf = static.input_text_buf[0..] });
+            _ = zgui.text("length of Input text {}", .{std.mem.len(@as([*:0]u8, static.input_text_buf[0..]))});
```

Comment:
these `@as` can be removed if https://github.com/ziglang/zig/pull/19542 is accepted